src/components/hero-section/hero-venue.astro

```diff
@@ -31,16 +31,23 @@ import venueImage from "./venue-hero.jpg";
       <slot />
     </Prose>
 
-    <div class="space-x-4" style={{ marginBottom: 10 }}>
+    <div class="space-x-4 mb-5">
       <ButtonLink href="/where#prague-congress-centre---conference-venue">
         PCC Conference Venue
       </ButtonLink>
-
       <Link
         href="https://www.google.com/maps?ll=50.062576,14.429021&z=14&t=m&mapclient=embed&cid=5559277398893748727"
```

Comment:
```suggestion
        href="https://maps.app.goo.gl/jXtn7oXVg6vECFNP9"
```

Let's shorten this

Reply:
Hmm now thinking about it again, it it better to link directly to Entrance 5? https://goo.gl/maps/S13ZYR1ktN7ebwyP7

Reply:
Yes, makes better sense. Thank you!

Reply:
@hypha updated the suggestion

Reply:
Oops,I wish I waited.  I just did it here: https://github.com/EuroPython/website/pull/762/commits/ebba04c38400840c183c0aa714b3f0759e582fab. Thank you so much though!